mxcubeweb/core/components/samplechanger.py

```diff
@@ -459,6 +481,10 @@ def queue_mount_sample(view, data_model, centring_done_cb, async_result):  # noq
                     "Sample changer could not load sample", ""
                 )
 
+    # Harvest Next sample while loading current
+    if mount_from_harvester and not HWR.beamline.harvester.get_room_temperature_mode():
```

Comment:
Its neater to do this with a signal on the Harvester `HardwareObject`, so that when a sample is mounted you trigger the harvesting of the following one.